src/test/groovy/com/vanniktech/android/apk/size/ApkSizeToolsSpec.groovy

```diff
@@ -1,42 +1,45 @@
 package com.vanniktech.android.apk.size
 
 import spock.lang.Specification
+import spock.lang.Unroll
 
 final class ApkSizeToolsSpec extends Specification {
-    Locale defaultLocale
-
-    def 'setup'() {
-        defaultLocale = Locale.default
-    }
+    def defaultLocale = Locale.default
 
     def 'cleanup'() {
-        Locale.setDefault(defaultLocale)
+        Locale.setDefault defaultLocale
     }
 
-    def 'convert bytes to mega bytes'() {
+    @Unroll
+    def 'convert bytes to mega bytes with #locale locale'() {
```

Comment:
Unroll will display each test case individually. Try running these tests in your IDE.